modules/core/src/utils/json-loader.js

```diff
@@ -0,0 +1,12 @@
+function isJSON(text) {
+  const firstChar = text[0];
+  const lastChar = text[text.length - 1];
+  return (firstChar === '{' && lastChar === '}') || (firstChar === '[' && lastChar === ']');
+}
+
+export default {
+  name: 'JSON',
+  extensions: ['json', 'geojson'],
```

Comment:
Somehow one would want to define two separate loaders for those two extensions, but this makes sense for now.